test/extractor.findKeys.test.ts

```diff
@@ -67,4 +75,199 @@ describe('extractor.findKeys', () => {
     expect(title?.ns).toBe('translation')
     expect(desc?.ns).toBe('translation')
   })
+
+  it('should call plugin onVisitNode hooks for each AST node', async () => {
```

Comment:
These tests fit better here now because `onVisitNode` is not called in the mocked out implementations in the `processFile` tests.